events.html

```diff
@@ -38,10 +39,10 @@
               <a class="nav-link js-scroll-trigger" href="about.html">About</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link js-scroll-trigger" href="events.html" style="color:black;">Events</a>
+              <a class="nav-link js-scroll-trigger" href="events.html">Events</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link js-scroll-trigger" href="resources.html" >Resources</a>
+              <a class="nav-link js-scroll-trigger" href="resources.html" style="color:black;">Resources</a>
```

Comment:
Why'd you change this? This basically showed which page was open and now its misleading and changes resources to black, when you are on events
<img width="1439" alt="screen shot 2018-08-12 at 11 42 52 am" src="https://user-images.githubusercontent.com/20584149/44003697-0742fd7e-9e25-11e8-8c1a-4933256f52cc.png">